rest-api/src/main/resources/definitions/upload.yml

```diff
@@ -9,82 +9,67 @@ required:
 properties:
     uploadId:
         type: string
-        readOnly: true
```

Comment:
This is part of our public API and used by clients. Make this readOnly. Use mock in unit tests. (This and all the other fields in this class.)